data/prixa-tech.json

```diff
@@ -0,0 +1,16 @@
+{
+ "name": "Prixa Technologies",
+ "establishedAt": 1446014709,
+ "location": "Jawalakhel, Lalitpur, Nepal",
+ "website": "https://prixa.net",
+ "email": "info@prixa.org",
+ "phone": "9801970966",
+ "socials": {
+   "facebook": "https://www.facebook.com/prixatech",
+   "github": null,
+   "instagram": null,
+   "linkedin": "https://www.linkedin.com/company/prixa",
+   "twitter": "https://twitter.com/prixatech",
+   "youtube": null
+ }
+}
```

Comment:
It seems there is a new line at the end of the file in other json files. Let's follow the same format.